pynestml/meta_model/ast_variable.py

```diff
@@ -144,6 +149,19 @@ def set_type_symbol(self, type_symbol):
             '(PyNestML.AST.Variable) No or wrong type of type symbol provided (%s)!' % type(type_symbol)
         self.type_symbol = type_symbol
 
+    def get_vector_parameter(self) -> str:
```

Comment:
Can the return type be either str or int here?

Reply:
It will be str because the text of the vector parameter token (for both integer and variable) is stored in `vector_parameter`.